openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -2222,6 +2223,16 @@ impl<T: CpuIo> hv1_hypercall::VtlCall for UhHypercallHandler<'_, '_, T, SnpBacke
     }
 }
 
+impl<T: CpuIo> hv1_hypercall::VtlReturn for UhHypercallHandler<'_, '_, T, SnpBacked> {
```

Comment:
Why do we keep doing this pattern instead of just implementing the hypercall for a generic backing `B`?

Reply:
I think this should at least be on the HardwareIsolatedBacking, which I missed. Why the generic one? Are you anticipating a future where openhcl will handle all vtl calls and returns instead of the hypervisor, even for  non-CVMs?

Reply:
No, it's OK to bound `B: HardwareIsolateedBacking`. I just want to avoid these wrapper impls.